nsis/SharedDefs.nsh

```diff
@@ -17,7 +17,7 @@ Var UserPrivileges
 Var IsAdmin
 
 !ifndef APP_VERSION
-  !define APP_VERSION      "1.6.0"
+  !define APP_VERSION      "1.8.0"
```

Comment:
For 1.8, the OS version check can be updated to reflect the minimum requirement of Windows 8.1 or greater - https://github.com/PCSX2/pcsx2/pull/3742/files#diff-25f3166319cb3cfbcbc8aed25acc2db8R45